src/EzSystemsPlatformHttpCacheBundle.php

```diff
@@ -15,5 +17,35 @@ public function build(ContainerBuilder $container)
 
         $container->addCompilerPass(new ResponseTaggersPass());
         $container->addCompilerPass(new KernelPass());
+        $container->addCompilerPass(new DriverPass());
+    }
+
+    public function getContainerExtension()
+    {
+        // We need to overload Bundle::getContainerExtension() because we change DI alias in
+        // EzSystemsPlatformHttpCacheExtension::getAlias()
+        // Bundle::getContainerExtension() has a check that alias matches underscored version of the bundle name
+        if (null === $this->extension) {
+            $extension = $this->createContainerExtension();
+
+            if (null !== $extension) {
+                if (!$extension instanceof ExtensionInterface) {
+                    throw new \LogicException(sprintf('Extension %s must implement Symfony\Component\DependencyInjection\Extension\ExtensionInterface.', get_class($extension)));
+                }
+
+                $this->extension = $extension;
+            } else {
+                $this->extension = false;
+            }
+        }
+
+        if ($this->extension) {
+            return $this->extension;
+        }
+    }
```

Comment:
Why can't we just `return new EzPlatformHttpCacheExtension()`? See [Symfony doc](https://symfony.com/doc/current/bundles/extension.html#manually-registering-an-extension-class)

Reply:
ah.... man.... This we talked about yesterday
Ok, I'll change it again

Reply:
fixed

Reply:
Sorry, I think I missed the point of that discussion and didn't see this code yesterday. Doing too many things at the same time ;)